app/javascript/mastodon/is_mobile.ts

```diff
@@ -16,10 +16,10 @@ export const layoutFromWindow = (): LayoutType => {
   }
 };
 
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-expect-error
 const iOS =
-  /iPad|iPhone|iPod/.test(navigator.userAgent) && window.MSStream != null;
+  /iPad|iPhone|iPod/.test(navigator.userAgent) &&
+  // @ts-expect-error MSStream is a legacy vendor property
+  window.MSStream != null;
```

Comment:
Should we still care about IE11? Its not supported by our browserslist and I think our JS no longer runs on it.

Reply:
It's used exactly… nowhere 😆

Reply:
I guess the `userTouching` is, but not the iOS stuff